lib/internal/bootstrap/node.js

```diff
@@ -244,7 +244,8 @@ Object.defineProperty(process, 'features', {
     tls_alpn: hasOpenSSL,
     tls_sni: hasOpenSSL,
     tls_ocsp: hasOpenSSL,
-    tls: hasOpenSSL
+    tls: hasOpenSSL,
+    cached_builtins: config.hasCachedBuiltins,
```

Comment:
Since this API has not been well maintained: is this really a good place to add this information? And is it actually valuable for our users?

Reply:
Less maintained, yes, but it's is correct. Also it's the only place we expose "computed" compile-time flags (`process.config` is static config-time values).

tl;dr it's the place that makes most sense without inventing a new one. Also @joyeecheung and I tentatively "plan" to move some more values from `process.config` to here.

Reply:
> Also it's the only place we expose "computed" compile-time flags (process.config is static config-time values).

We actually have `internalBinding('config')` that basically is dedicated to this..

Reply:
Well the `config` in `config.hasCachedBuiltins` _is_ `internalBinding('config')` 😉 

But IMHO this is an interesting enough value to "expose as public". It does allow the user to do some introspection about the binary